po/de_DE.UTF-8.po

```diff
@@ -0,0 +1,253 @@
+#: src/apprt/gtk/ui/1.0/menu-window-titlebar_menu.blp:90
+msgid "Open Configuration"
+msgstr "Konfiguration öffnen"
+
+#: src/apprt/gtk/ui/1.0/menu-surface-context_menu.blp:97
+#: src/apprt/gtk/ui/1.0/menu-window-titlebar_menu.blp:95
+msgid "Reload Configuration"
+msgstr "Konfiguration neu laden"
+
+#: src/apprt/gtk/ui/1.0/menu-window-titlebar_menu.blp:85
+msgid "Terminal Inspector"
+msgstr "Terminalinspektor"
+
+#: src/apprt/gtk/ui/1.0/menu-window-titlebar_menu.blp:102
+#: src/apprt/gtk/Window.zig:933
+msgid "About Ghostty"
+msgstr "Über Ghostty"
+
+#: src/apprt/gtk/ui/1.0/menu-window-titlebar_menu.blp:107
+msgid "Quit"
+msgstr "Beenden"
+
+#: src/apprt/gtk/ui/1.5/ccw-osc-52-read.blp:6
+#: src/apprt/gtk/ui/1.5/ccw-osc-52-write.blp:6
+msgid "Authorize Clipboard Access"
+msgstr "Zugriff auf die Zwischenablage gewähren"
+
+#: src/apprt/gtk/ui/1.5/ccw-osc-52-read.blp:7
+msgid ""
+"An application is attempting to read from the clipboard. The current "
+"clipboard contents are shown below."
+msgstr ""
+"Eine Anwendung versucht von der Zwischenablage zu lesen. Der aktuelle Inhalt "
+"der Zwischenablage wird unten angezeigt."
+
+#: src/apprt/gtk/ui/1.5/ccw-osc-52-read.blp:10
+#: src/apprt/gtk/ui/1.5/ccw-osc-52-write.blp:10
+msgid "Deny"
+msgstr "Nicht erlauben"
+
+#: src/apprt/gtk/ui/1.5/ccw-osc-52-read.blp:11
+#: src/apprt/gtk/ui/1.5/ccw-osc-52-write.blp:11
+msgid "Allow"
+msgstr "Erlauben"
+
+#: src/apprt/gtk/ui/1.5/ccw-osc-52-write.blp:7
+msgid ""
+"An application is attempting to write to the clipboard. The current "
+"clipboard contents are shown below."
+msgstr ""
+"Eine Anwendung versucht in die Zwischenablage zu schreiben. Der aktuelle "
+"Inhalt der Zwischenablage wird unten angezeigt."
+
+#: src/apprt/gtk/ui/1.5/ccw-paste.blp:6
+msgid "Warning: Potentially Unsafe Paste"
+msgstr "Achtung: Möglicherweise unsicheres Einfügen"
+
+#: src/apprt/gtk/ui/1.5/ccw-paste.blp:7
+msgid ""
+"Pasting this text into the terminal may be dangerous as it looks like some "
+"commands may be executed."
+msgstr ""
+"Diesen Text in das Terminal einzufügen könnte möglicherweise gefährlich "
+"sein. Es scheint, dass Anweisungen ausgeführt werden könnten."
+
+#: src/apprt/gtk/Window.zig:199
+msgid "Main Menu"
+msgstr "Hauptmenü"
+
+#: src/apprt/gtk/Window.zig:219
+msgid "View Open Tabs"
+msgstr "Offene Tabs einblenden"
+
+#: src/apprt/gtk/Window.zig:265
+msgid ""
+"⚠️ You're running a debug build of Ghostty! Performance will be degraded."
+msgstr ""
+"⚠️ Du verwendest einen Debug Build von Ghostty! Die Leistung wird reduziert "
+"sein."
+
+#: src/apprt/gtk/Window.zig:681
+msgid "Reloaded the configuration"
+msgstr "Konfiguration wurde neu geladen"
+
+#: src/apprt/gtk/Window.zig:914
+msgid "Ghostty Developers"
+msgstr "Ghostty-Entwickler"
+
+#: src/apprt/gtk/CloseDialog.zig:47
+#, fuzzy
+msgid "Close"
+msgstr "Schließen"
+
+#: src/apprt/gtk/CloseDialog.zig:87
+#, fuzzy
+msgid "Quit Ghostty?"
+msgstr "Über Ghostty?"
```

Comment:
These lines are automatically "guessed" by gettext (see the fuzzy flag) and haven't actually been translated